package.json

```
@@ -1,12 +1,12 @@
 {
     "author": {
-        "name": "Edrean Ernst"
+        "name": "Edrean Ernst;Pedro Torrezão"
     },
     "bugs": {
         "url": "https://github.com/edreanernst/node-red-contrib-opsgenie/issues"
     },
     "name": "node-red-contrib-opsgenie",
-    "version": "0.1.0",
+    "version": "0.1.1",
```

I will change the version later
```suggestion
    "version": "0.1.0",
```